src/sc.spatialfeature.js

```diff
@@ -16,7 +16,7 @@ export function spatialFeature(storeId, layerId, obj) {
     type: 'Feature'
   };
 
-  spatialFeature = _.defaults(defaultObj, obj);
+  spatialFeature = _.defaults(obj, defaultObj);
```

Comment:
Yes I was confused by this. Isn't `obj` what we want to mutate? We want to apply all of the defaults to `obj` if it doesn't already have those. So if we pass in `{ properties: { name: 'Tito' } }` we want an object that has those defaults and `{ name: 'Tito' }` properties. Since the default obj properties is `{}` we wouldn't be able to assign that to the new obj. I made a jsfiddle to demonstrate https://jsfiddle.net/xmcz9pc4/3/